bin/node/src/aleph_cli.rs

```diff
@@ -3,9 +3,12 @@ use std::path::PathBuf;
 use aleph_primitives::DEFAULT_UNIT_CREATION_DELAY;
 use clap::{ArgGroup, Parser};
 use finality_aleph::UnitCreationDelay;
+use sc_service::Configuration;
+
+const DEFAULT_BACKUP_FOLDER: &str = "backup-stash";
```

Comment:
If you make this a constant, maybe `bootstrap-chain` should use it too? (struct `ChainParams` in `chain_spec.rs`)

Reply:
done